src/metalnx-web/src/main/resources/i18n/messages_en.properties

```diff
@@ -508,6 +508,7 @@ collections.form.no.access.error=You do not have rights to create collection {0}
 
 ## Collection modification form
 collections.edit.form.title=Modify Collection
+collection.rename.form.title=Rename Collection
```

Comment:
the others are plural.... `collections.`  

should this be the same?

Reply:
right, let me change this